DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factory/IAnonymizerSettingsFactory.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    public interface IAnonymizerSettingsFactory
+    {
+        public T CreateAnonymizerSetting<T>(JObject settings);
```

Comment:
It seems this interface is never used. All references point to the implementation `AnonymizeSettingsFactory.CreateAnonymizerSetting`.

Reply:
Seems we can remove this.